amm/interp/src/main/scala/ammonite/runtime/Interpreter.scala

```diff
@@ -699,9 +700,17 @@ object Interpreter{
     Name(wrapperName.raw + (if (wrapperIndex == 1) "" else "_" + wrapperIndex))
   }
 
-  def initPrinters(output: OutputStream, error: OutputStream) = {
+  def initPrinters(output: OutputStream, error: OutputStream, verboseOutput: Boolean) = {
     val colors = Ref[Colors](Colors.Default)
-    val printStream = new PrintStream(output, true)
+    val printStream = verboseOutput match{
+      case true => new PrintStream(output, true)
+      //make a dummy printstream in case of non-verbose output
+      case false => new PrintStream(new OutputStream(){
+        @Override
+        def write(b: Int){
+        }
```

Comment:
Skip the `@Override` annotation (not necessary in Scala) and let's define write as `= ()` since the `(b: Int){}` syntax is getting deprecated